extensions/VisualEditor/wikia/modules/ve/init/ve.init.wikia.ViewPageTarget.init.js

```diff
@@ -229,7 +229,7 @@
 
 			if ( history.pushState && uri.query.veaction !== 'edit' ) {
 				if (!pageExists) {
```

Comment:
What do you think about moving this if statement to the place where hook is invoked?